exercises/practice/reverse-string/.meta/solutions/reverse_string.rb

```diff
@@ -0,0 +1,5 @@
+module ReverseString
```

Comment:
We might name it similar to other modules in Ruby, with "able" at the end.  But we do not have to.